src/plugins/experimental-discord/plugin.js

```diff
@@ -66,16 +67,24 @@ export class DiscordPlugin implements Plugin {
     rd: ReferenceDetector
   ): Promise<WeightedGraph> {
     const _ = rd; // TODO(#1808): not yet used
-    const {guildId, reactionWeights, roleWeightConfig} = await loadConfig(ctx);
+    const {
+      guildId,
+      reactionWeights,
+      roleWeightConfig,
+      channelWeightConfig,
+    } = await loadConfig(ctx);
     const repo = await repository(ctx, guildId);
     const declarationWeights = weightsForDeclaration(declaration);
     const defaultRoleWeightConfig = {defaultWeight: 1, roleWeights: {}};
+    const defaultChannelWeightConfig = {defaultWeight: 1, channelWeights: {}};
+
     return await createGraph(
       guildId,
       repo,
       declarationWeights,
       reactionWeights,
-      roleWeightConfig || defaultRoleWeightConfig
+      NullUtil.orElse(roleWeightConfig, defaultRoleWeightConfig),
+      NullUtil.orElse(channelWeightConfig, defaultChannelWeightConfig)
```

Comment:
I don't think this will quite work, but I'll resolve it in another PR, since it's a pre-existing issue.

Reply:
Nevermind! It's fine. :)